doc/api/n-api.md

```diff
@@ -1375,6 +1379,31 @@ This API allocates a `node::Buffer` object and initializes it with data copied
 from the passed-in buffer. While this is still a fully-supported data
 structure, in most cases using a TypedArray will suffice.
 
+#### napi_create_date
+<!-- YAML
+added: REPLACEME
+napiVersion: 4
```

Comment:
This shouldn't be set? https://github.com/nodejs/node/pull/28330

Reply:
We need to land this as is and then backport #28330.